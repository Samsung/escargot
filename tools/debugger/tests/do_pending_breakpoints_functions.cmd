b f
y
b b
y
b non-existing-breakingpoint
y
list
c
list
source
c
source
c
