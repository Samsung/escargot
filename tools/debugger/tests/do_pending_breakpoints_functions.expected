Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_pending_breakpoints_functions.js:20
(escargot-debugger) b f
No breakpoint found, do you want to add a pending breakpoint? (y or [n])
Pending breakpoint 1 at f()
(escargot-debugger) b b
No breakpoint found, do you want to add a pending breakpoint? (y or [n])
Pending breakpoint 2 at b()
(escargot-debugger) b non-existing-breakingpoint
No breakpoint found, do you want to add a pending breakpoint? (y or [n])
Pending breakpoint 3 at non-existing-breakingpoint()
(escargot-debugger) list
=== Pending breakpoints ===
 1: f() (pending)
 2: b() (pending)
 3: non-existing-breakingpoint() (pending)
(escargot-debugger) c
Print: pending-breakpoints_functions
Breakpoint 4 at eval input:2 (in f() at line:1, col:1)
Breakpoint 5 at eval input:2 (in b() at line:1, col:1)
Stopped at breakpoint:4 eval input:2 (in f() at line:1, col:1)
(escargot-debugger) list
=== Active breakpoints ===
 4: eval input:2 (in f() at line:1, col:1)
 5: eval input:2 (in b() at line:1, col:1)
=== Pending breakpoints ===
 3: non-existing-breakingpoint() (pending)
(escargot-debugger) source
Source: eval input
   1   function f()
   2 > { return 5 }
(escargot-debugger) c
Print: 5
Stopped at breakpoint:5 eval input:2 (in b() at line:1, col:1)
(escargot-debugger) source
Source: eval input
   1   function b()
   2 > { return 8 }
(escargot-debugger) c
Print: 8
Connection closed.
